integrationtests/server-integration/third-party-server/pom.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-server-integration</artifactId>
+        <version>12.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>infinispan-server-integrationtests-third-party</artifactId>
+
+    <properties>
+        <version.tomcat>9.0.37</version.tomcat>
+        <version.wildfly>20.0.1.Final</version.wildfly>
+        <version.arquillian-tomcat-embedded>1.1.0.Final</version.arquillian-tomcat-embedded>
```

Comment:
@pruivo should we move to build?
It seems so specific to move to a high-level project.

Reply:
it is ok. do we really need another wildfly version? :sob:

Reply:
Can we change https://github.com/infinispan/infinispan/blob/master/build-configuration/pom.xml#L114-L115 to something different?
* appserver.groupId: wildfly.groupId
* appserver.version: version.wildfly

Reply:
you can change it, but then you may have problems downstream since we build against EAP 7.3 (WF 18).

Reply:
ok, how about change only the property names ?

Reply:
what properties?

Reply:
from `appserver.groupId` to `wildfly.groupId`
from `appserver.version` to `version.wildfly`